test/StdChains.t.sol

```diff
@@ -17,25 +17,43 @@ contract StdChainsTest is Test {
 
     // Ensure we can connect to the default RPC URL for each chain.
     function testRpcs() public {
-        (string[2][] memory rpcs) = vm.rpcUrls();
-        for (uint256 i = 0; i < rpcs.length; i++) {
-            ( /* string memory name */ , string memory rpcUrl) = (rpcs[i][0], rpcs[i][1]);
-            vm.createSelectFork(rpcUrl);
-        }
+        string memory rpcUrl;
```

Comment:
We now only check an arbitrary subset of default URLs, but we should still test all our default RPCs. Can conditionally skip any aliases that we know have invalid ones and will revert